data/ecosystems/s/stacks.toml

```diff
@@ -5,6 +5,7 @@ sub_ecosystems = [
   "alexgo-io",
   "arkadiko-dao",
   "citycoins",
+  "CrossCheck",
```

Comment:
This needs to have an accompanying toml file

Reply:
I'll merge this and fix into #381